### DISCUSSION TITLE
Raw Sh1ms not available.

### DISCUSSION THREAD

**Discussion by SupremeRubisco** [3kh0/ext-remover#442] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://lenovo-driver-download.com/ doesn't have any raw shims available, can you change it to https://obelous.dev/sh1mmer/?

**Comment by Aka-but-nice** [comment]
I am not in charge of that. I'm pretty sure obelous is in the Titanium Network discord, which I am in too, but unaware of where his position is at the moment.

**Reply by SupremeRubisco** [reply]
Ok, I just thought people needed raw shims.

**Comment by 3kh0** [comment]
That was just a hint, you can find the shim files, what computer and board do you have?

**Reply by SupremeRubisco** [reply]
I don't know what computer but I have octopus.
